jetty-openid/src/main/java/org/eclipse/jetty/security/openid/OpenIdCredentials.java

```diff
@@ -126,7 +127,7 @@ private void validateClaims(OpenIdConfiguration configuration) throws Exception
             throw new AuthenticationException("ID Token has expired");
     }
 
-    private void validateAudience(OpenIdConfiguration configuration) throws AuthenticationException
+    private static void validateAudience(Map<String, Object> claims, OpenIdConfiguration configuration) throws AuthenticationException
```

Comment:
why static and passing in claims? why not keep as a non-static?  If the answer is for testing.... I'd rather keep the design correct... perhaps with a special constructor.

Reply:
I don't really think it makes the design incorrect, it makes sense to have a method which validates claims given a configuration. And this doesn't impact anything because these methods are not public. I could add a constructor for `OpenIdCredentials(Map<String, Object> claims)` but that makes less sense to me.

I'll give it a try and see if it makes it any better.